api/go.mod

```diff
@@ -4,16 +4,15 @@ go 1.18
 
 require (
 	github.com/openshift/api v0.0.0-20220124143425-d74727069f6f // release-4.10
-	k8s.io/apimachinery v0.23.5
+	k8s.io/apimachinery v0.25.15
 	kubevirt.io/containerized-data-importer-api v1.53.1
 	kubevirt.io/controller-lifecycle-operator-sdk/api v0.2.4
-	sigs.k8s.io/controller-runtime v0.11.2
+	sigs.k8s.io/controller-runtime v0.13.2
```

Comment:
The same question here.

Reply:
Same, oldest release branch with the fix is `v0.13.2`.